gradle.properties

```diff
@@ -19,6 +19,7 @@ org.gradle.caching=true
 org.gradle.parallel=true
 org.gradle.configureondemand=true
 org.gradle.jvmargs=-Xss10240k
+org.gradle.vfs.watch=true
```

Comment:
I'm a little wary of this. The [blog post](https://blog.gradle.org/introducing-file-system-watching) does say its ready for production use... but I'm wondering if we should just let individuals try it themselves until it's enabled by default.

Reply:
I trust the Gradle team when they say that it is ready for production use. They use it themselves in the Gradle build: https://github.com/gradle/gradle/blob/master/gradle.properties#L14

Reply:
Ok I'm fine with keeping it, it's easy enough to revert in the unlikely case it does end up causing issues.